project/src/main/world/creature/creature-visuals.gd

```diff
@@ -245,11 +231,7 @@ func set_orientation(new_orientation: int) -> void:
 		# orientation by making it something nonsensical
 		old_orientation = -1
 	
-	if $Animations/MovementPlayer.current_animation == "idle-nw" and new_orientation in [SOUTHEAST, SOUTHWEST]:
-		$Animations/MovementPlayer.play("idle-se")
-	elif $Animations/MovementPlayer.current_animation == "idle-se" and new_orientation in [NORTHEAST, NORTHWEST]:
-		$Animations/MovementPlayer.play("idle-nw")
-	
+	animations.update_orientation(new_orientation)
```

Comment:
Can animations react to the `orientation_changed` signal like everyone else?

Reply:
Changed animations to react to `orientation_changed` signal instead of having this function call.